docs/guides/dns/cloudflared.md

```diff
@@ -239,6 +239,12 @@ sudo chmod +x /etc/cron.weekly/cloudflared-updater
 sudo chown root:root /etc/cron.weekly/cloudflared-updater
 ```
 
+<!-- markdownlint-disable code-block-style -->
+!!! warning
+    Make sure to add shebang `#!/bin/bash` in the beginning of the file.
```

Comment:
Thanks for your PR.

Do you think it would be enough to change the text to

```
If you want to have the system update cloudflared automatically, simply place the update commands for your configuration method in a bash script at /etc/cron.weekly/cloudflared-updater, and adjust permissions:
```

to make users aware that a shebang is needed?
___

Otherwise I would suggest

```suggestion
    Make sure to add shebang `#!/bin/bash` in the beginning of the `/etc/cron.weekly/cloudflared-updater`.
```